db/migrations/0018_add_lti_usage.py

```diff
@@ -0,0 +1,43 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement_types (
+            id INTEGER NOT NULL AUTO_INCREMENT,
+            description VARCHAR(100) NOT NULL,
+            table_name VARCHAR(64) NOT NULL,
```

Comment:
I'm not sure this `table_name` is needed. Need to think more about this.

Reply:
I have this in here because we either have the table names in the code or we have this in the database.

This entire table could be in the code too instead of the database.

Reply:
I'm somewhat leaning on deleting this table and just doing the mapping in the code and when/if we add placement(s) other than Zoom doing it then.

Reply:
Yeah, so far, I've taken the somewhat lazy approach of not defining controlled vocabularies in the database (for enrollment types, for example). I kind of figured that this was an intermediary data source, and that the DWing team would follow best practices to create an analytical dataset.

Regarding the `table_name`, yeah, it might make more sense to include this in the codebase as an `enum`. I'm just thinking that the `table_name` here is describing a related entity (a Zoom meeting, a ProctorU test, etc.), and thus it feels a little out of place in this `lti_placement_type`.